fecgov/fec-cms
# Feature/5816 debts line number filter

## Summary

- Resolves #5816 

This should merge the previously-reviewed line number filters into Debts

### Required reviewers

- ux
- Front-end?

## Impacted areas of the application

The /data/debts tables & details

## Screenshots

(Include a screenshot of the new/updated features in context (“in the wild”). If it is an interface change, include both before and after screenshots)

## Related PRs



## How to test

- pull the branch
- `npm run test-single` to make sure tests are good
- `npm run build`
- check the [Debts table](http://127.0.0.1:8000/data/debts/)
